srcpkgs/tremc/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'tremc'
+pkgname=tremc
+version=0.9.1
+revision=1
+wrksrc="${pkgname}-${version}"
+build_style=gnu-makefile
+hostmakedepends="python3 git"
+makedepends="python3"
+depends="python3"
+short_desc="A console client for the BitTorrent client Transmission"
+maintainer="Nathan D <nathan.dhanas@gmail.com>"
+license="GPL-3.0"
+homepage="https://github.com/tremc/tremc"
+
+do_fetch() {
```

Comment:
Nope. You should use the release tarball `https://github.com/tremc/tremc/archive/0.9.1.tar.gz`